src/js/components/Breadcrumb.js

```diff
@@ -46,6 +72,12 @@ class Breadcrumb extends React.Component {
   }
 
   shouldComponentUpdate(nextProps, nextState) {
+    let {buildCrumbCustomParams} = this.props;
+
+    if (!DeepEqual(nextProps.buildCrumbCustomParams, buildCrumbCustomParams)) {
```

Comment:
I thought there might have been a reason you had `nextState.availableWidth == null || nextState.expandedWidth == null ` first, to circuit some process?